tests/unit/config/test_configfiles.py

```diff
@@ -1000,6 +1000,28 @@ def test_binding_options_hidden(self):
         assert 'bindings.default' not in text
         assert 'bindings.commands' not in text
 
+    def test_unbind(self):
+        opt = configdata.Option(
+            name='opt', typ=configtypes.Int(), default='def',
+            backends=[usertypes.Backend.QtWebEngine], raw_backends=None,
+            description='Hello World')
+        options = [(None, opt, 'val')]
```

Comment:
This doesn't seem to be required for what you actually want to test - you're just interested in `config.unbind(...)` (and maybe a `config.bind`) working correctly, so I think you could just pass `options=[]` to `ConfigPyWriter` for this test to simplify it.